test/conftest.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+
+from test.pylib.report_plugin import ReportPlugin
+
+ALL_MODES = {'debug': 'Debug',
+             'release': 'RelWithDebInfo',
+             'dev': 'Dev',
+             'sanitize': 'Sanitize',
+             'coverage': 'Coverage'}
+
+def pytest_addoption(parser):
+    parser.addoption('--mode', choices=ALL_MODES.keys(), dest="mode",
+                     help="Run only tests for given build mode(s)")
+    parser.addoption('--tmpdir', action='store', default='testlog', help='''Path to temporary test data and log files. The data is
+            further segregated per build mode. Default: ./testlog.''', )
+    parser.addoption('--repeat', action='store', type=int, help='Number of times to repeat each test')
```

Comment:
If you're not using some functionality, it doesn't mean it useless. As far as I remember, @kbr-scylla is using it. 
We can't. This will not work for boost and unit, since they are neither python tests nor applying fixtures. So we cannot use plugin.

Reply:
Where is @kbr-scylla use it?
I don't understand... This adds an option called "repeat". To pytest. Where is it used in pytest?
I know that **test.py** has a "--repeat" option. But that's not what you're adding here.

It's easy to answer "If you're not using some functionality, it doesn't mean it useless.", but sometimes some code **is** useless. Please convince me that this one isn't useless (in conftest.py, NOT in test.py).

Reply:
You are right, I've mistaken the PR's. This is from another one. I'll delete this, and add it later during merge conflict.